ansible/roles/node/tasks/main.yml

```diff
@@ -12,7 +12,7 @@
 - name: install node
   become: false
   shell: >
-    . {{ ansible_env.HOME }}/.nvm/nvm.sh && nvm install v8
+    . {{ ansible_env.HOME }}/.nvm/nvm.sh && nvm install && nvm use
```

Comment:
@DanielBadura this breaks (at least initial) provisioning 

`stderr": "No .nvmrc file found\n\nNode Version Manager\n\nNote: <version> refers to any version-like string nvm understands`